corehq/apps/fixtures/dbaccessors.py

```diff
@@ -29,15 +29,15 @@ def get_fixture_data_types_in_domain(domain):
     ))
 
 
-@quickcache(['domain', 'data_type_ids'], timeout=60 * 60, memoize_timeout=60, skip_arg='bypass_cache')
-def get_fixture_items_for_data_types(domain, data_type_ids, bypass_cache=False):
+@quickcache(['domain', 'data_type_id'], timeout=60 * 60, memoize_timeout=60, skip_arg='bypass_cache')
+def get_fixture_items_for_data_type(domain, data_type_id, bypass_cache=False):
     from corehq.apps.fixtures.models import FixtureDataItem
     return list(FixtureDataItem.view(
         'fixtures/data_items_by_domain_type',
-        keys=[[domain, id] for id in data_type_ids],
+        startkey=[domain, data_type_id],
+        endkey=[domain, data_type_id, {}],
```

Comment:
Note that the old key format is not forward compatible.  Calls to this between migration and code deploy will fail.  Might end up making an accommodation for that if necessary